SecureSubmit.php

```diff
@@ -801,7 +801,7 @@ function shortcode($atts) {
                 var <?php echo $prefix; ?>_card_html = "<div id='<?php echo $prefix; ?>_card_panel'>";
                 <?php echo $prefix; ?>_card_html += "<div class='back-button'><a href='#shipping' id='<?php echo $prefix; ?>_card_back'>back</a></div>";
                 <?php echo $prefix; ?>_card_html += "<div class='checkout-card-information'>Card Information</div>";
-                <?php echo $prefix; ?>_card_html += "<div class='card-number'><input type='text' id='card_number' class='checkout-input checkout-card required' placeholder='4111 - 1111 - 1111 - 1111'></div>";
+                <?php echo $prefix; ?>_card_html += "<div class='card-number'><input type='text' id='card_number' class='checkout-input checkout-card required' placeholder='Credit Card'></div>";
```

Comment:
I'd prefer to keep the status quo, but we need to get a fix pushed out for our developers/merchants. We can circle back for a solution here